packages/node_modules/@webex/plugin-meetings/src/meeting/util.js

```diff
@@ -744,4 +744,40 @@ MeetingUtil.handleDeviceLogging = (devices = []) => {
   });
 };
 
+MeetingUtil.endMeetingForAll = (meeting, options = {}) => {
+  if (meeting.meetingState === FULL_STATE.INACTIVE) {
+    // TODO: clean up if the meeting is already inactive
```

Comment:
we shouldn't have any todo comments in the code - if this needs to be done, it should be done in this PR or if for some good reason cannot be done now, raise a jira for tracking. (I know this is a copy&paste from the leave() method, but we should still try to do something about it as we're touching this area)

Reply:
cleaned the todo comment

Reply:
cleaned the todo comment